src/main/kotlin/org/jetbrains/kotlin/konan/util/DefFile.kt

```diff
@@ -140,12 +139,11 @@ private fun parseDefFile(file: File?, substitutions: Map<String, String>): Tripl
          propertyLines = lines
          headerLines = emptyList()
      }
-
-     val propertiesReader = StringReader(propertyLines.joinToString(System.lineSeparator()))
+     val propertiesReader = StringReader(propertyLines.joinToString(System.lineSeparator()).replace("\\\\(?=.)".toRegex(), "\\\\\\\\"))
```

Comment:
May be we should just go with quotation alone, without backslash support. Eight backslashes does not conform to any definition of maintainable code.

Reply:
It may be worth using a raw string (like `"""\\\\"""`) to avoid having eight backslashes. 

BTW why do we have 8 of them in this source (== 4 in the replacement)? Don't we need to just double backslashes in the input file (== have 2 backslashes in the replacement)?

Reply:
We need to replace \ to \\, because properties reader also use \ for escaping. Here something else eats extra \, may be StringReader.

I think we need \, because then there can be problens with window pathes for example.

Reply:
> because then there can be problens with window pathes for example.

What do you mean?

Reply:
Windows uses \ in pathes, if I don't replace it to repeated twice, it will be removed by properties loader

Reply:
Moreover, I think that there will be cases when quotes have to be inside quotes. As now necessity of spaces appears. It's better to do right now.